### ISSUE TITLE
Qutebrowser always crash from specific wayland compositor (labwc).

### ISSUE THREAD

**Issue by ahmadraniri1994** [qutebrowser/qutebrowser#7012]
**Version info**: 
> qutebrowser v2.4.0
> Git commit: 
> Backend: QtWebEngine 5.15.8, based on Chromium 87.0.4280.144
> Qt: 5.15.2

**Does the bug happen if you start with `--temp-basedir`?**: yes

**Description**
When I open qutebrowser and start to browse with it, it suddenly become hang for the first 2 minutes or less and then crash and force closed (from terminal I get segmentation fault). It happens specifically on labwc wayland compositor. I contact their developer and they didn't face same problem like me, I'm getting same behaviour on my two setups (Archlinux and debian, both with labwc wayland compositor.
**How to reproduce**
- Getting stracktrace with gdb, this is the gdb.txt : 
[gdb.txt](https://github.com/qutebrowser/qutebrowser/files/8030979/gdb.txt)

**Comment by The-Compiler** [comment]
The crash happens deep inside Qt's wayland code, so I doubt there is much qutebrowser can do about it. Can you try with another browser based on Qt 5 and QtWebEngine, such as KDE Falkon?

**Comment by ahmadraniri1994** [comment]
> Can you try with another browser based on Qt 5 and QtWebEngine, such as KDE Falkon?
Yeah, I tried falkon too and face same issue. Other wayland compositors seems don't face it.

**Comment by The-Compiler** [comment]
Clearly seems like an issue in the compositor then, or perhaps something between Qt and the compositor. Don't think there is anything qutebrowser could do about it. Perhaps you could try the `qt6-test` branch too, see https://github.com/qutebrowser/qutebrowser/issues/5395#issuecomment-964569192.

**Comment by ahmadraniri1994** [comment]
Thanks.

**Comment by ahmadraniri1994** [comment]
Turns out it's triggered by mouse or touchpad scroll and it's probably compositor things. I tried to use qutebrowser without mOuse and touchpad and it's fine.
